www/front_src/src/Resources/Graph/Performance/AnomalyDetection/ModalAD.tsx

```diff
@@ -0,0 +1,94 @@
+import { Dispatch, SetStateAction } from 'react';
+
+import { useUpdateAtom } from 'jotai/utils';
+
+import Modal from '@mui/material/Modal';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import makeStyles from '@mui/styles/makeStyles';
+import Paper from '@mui/material/Paper';
+
+import TimePeriodButtonGroup from '../TimePeriods';
+import ExportablePerformanceGraphWithTimeline from '../ExportableGraphWithTimeline';
+import { Resource } from '../../../models';
+import { ResourceDetails } from '../../../Details/models';
+
+import { openModalADAtom } from './anomalyDetectionAtom';
+
+interface Props {
+  details: Resource | ResourceDetails;
+  isOpen: boolean;
+  setIsOpen: Dispatch<SetStateAction<boolean>>;
+}
+
+const useStyles = makeStyles((theme) => ({
+  close: {
+    alignItems: 'flex-end',
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+  container: {
+    backgroundColor: theme.palette.background.default,
+    left: '50%',
+    padding: theme.spacing(2),
+    position: 'absolute',
+    top: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: '90%',
+  },
```

Comment:
You do you need to use the position absolute?

Reply:
with the relative position, I have to find the right values ​​of margin Top and left , with absolute position the modal is well centered (following their example from the doc) 🤔

Reply:
Is possible to test the display flex solution using justify and align properties ?